pkg/unvanquished_src.dpkdir/bots/build.bt

```diff
@@ -29,7 +29,8 @@ selector
 				{
 					behavior subroutine_become_builder_humans
 
-					condition distanceTo( E_H_REACTOR ) <= 700
+					// if reactor is close or does not exist
+					condition distanceTo( E_H_REACTOR ) <= 700 || distanceTo( E_H_REACTOR ) > 999999
```

Comment:
I would suggest:

 ```suggestion
					condition numOurBuildings( E_H_REACTOR ) == 0 || distanceTo( E_H_REACTOR ) <= 700
```

And likewise for the other 3 conditions.

For the rationale, sounds like a very good idea, thanks !

Reply:
`numOurBuildings` is slower than calculating the distance, as it loops over entities. I added a fast track for spawns, but that is the only fast case for `numOurBuildings`.

Reply:
hmm, ok, fair enough. I think since it has a comment at least the intent is documented. LGTM without test from me

Reply:
numOurBuildings does need to be used so that under-construction buildables are correctly detected.